### PR TITLE
SAP Hana Tests

### PR DESCRIPTION
**Pull request by rockambole86**
### Background

Since Net Core DLL for SAP Hana is not confirmed to work 100% with Linux/Mac, some connection tests are needed to determine if the DLL is working or not.

### Changes

Added a method with Try/Catch in order to determine if there is an error while connecting to database.

### CODE REVIEW

File: Billing.API/Services/Invoice/InvoiceService.cs [hunk 1]
```diff
@@ -75,9 +75,32 @@ public async Task<IEnumerable<InvoiceListItem>> GetInvoices(string clientId)
...
+
+                var dt = new DataTable("Invoices");
+
+                da.Fill(dt);
+
+                await conn.CloseAsync();
+            }
+            catch (Exception e)
+            {
+                return e.Message;
```
Comment (nreal) [inline Billing.API/Services/Invoice/InvoiceService.cs:100]: What do you think  `add a _logger.error(e, "Error connecting with Hana.")`?

Comment (rockambole86) [reply Billing.API/Services/Invoice/InvoiceService.cs:100]: I hand't thought about checking the log. thanks. BTW, I need the specific error... in order to determine if the problem is with the DLL or VPN related

Comment (andresmoschini) [reply Billing.API/Services/Invoice/InvoiceService.cs:100]: So, you will add the log, right?

Comment (rockambole86) [reply Billing.API/Services/Invoice/InvoiceService.cs]: Just added it :D

File: Billing.API/Services/Invoice/InvoiceService.cs [hunk 2]
```diff
@@ -75,9 +75,34 @@ public async Task<IEnumerable<InvoiceListItem>> GetInvoices(string clientId)
...
+
+                var dt = new DataTable("Invoices");
+
+                da.Fill(dt);
+
+                await conn.CloseAsync();
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError(ex.Message);
```
Comment (andresmoschini) [inline Billing.API/Services/Invoice/InvoiceService.cs:100]: Please, log _the error_, not the message, in the contrary you will loose information.

```suggestion
                _logger.LogError(ex);
```

Comment (andresmoschini) [reply Billing.API/Services/Invoice/InvoiceService.cs:100]: And better yet, the exception and a message as @nreal suggested: 

```suggestion
                _logger.error(ex, "Error testing Hana connection.");
```

**Pull request merged**